src/content/blog/astro-510.mdx

```diff
@@ -0,0 +1,151 @@
+
+```astro
+---
+import CartButton from '../components/CartButton.astro';
+---
+<CartButton server:defer />
+```
+
+### Drivers
+
+Astro sessions are powered by [unstorage](https://unstorage.unjs.io/) drivers, which are modules that Astro uses to store the session data. Unstorage supports dozens of providers, giving you a world of choice for storing the user data.
+
+ When sessions are a stable feature, Astro's official adapters will automatically configure a default driver. But during this experimental phase you must configure a `driver` for your chosen adapter yourself. You can also use a cross-platform driver such as [Upstash](https://unstorage.unjs.io/drivers/upstash) or [Redis](https://unstorage.unjs.io/drivers/redis).
+
+### Usage
+
+To try experimental sessions, add the `experimental.sessions` flag to your `astro.config.mjs`:
+
+This example shows how to enable the filesystem driver, which works with the Node.js adapter:
+
+```js title="astro.config.mjs" ins={4-8}
+  {
+    adapter: node({ mode: 'standalone' }),
+    experimental: {
+      session: {
+        // Required: the name of the Unstorage driver
+        driver: "fs",
+      },
+    },
+  }
+```
+
+For more information see [the sessions docs for this experimental flag ](https://docs.astro.build/en/reference/experimental-flags/sessions/). We'd also appreciate your feedback on the feature. For even more details, and to leave feedback and participate in the development of the feature, see [the Sessions RFC](https://github.com/withastro/roadmap/pull/1055).
+
+## Remote image caching
+
+You already know and love the built-in `<Image />` component to [display optimized images](https://docs.astro.build/en/guides/images/#display-optimized-images-with-the-image--component) with automatic resizing and optimization. While this component currently transforms both local and remote images, image caching between builds was limited to local images. 
+
+Now, Astro will cache remote images as well, reducing the need to re-download and re-transform images on every build. It uses [conditional requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Conditional_requests#conditional_headers) with [`If-Modified-Since`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since) or [`If-None-Match`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match) headers to determine whether the image is still fresh, and will use the cached version unless the source image has changed. 
+
+This is now enabled for all sites, and will work with any host that supports conditional requests. See the new [asset caching docs](https://docs.astro.build/en/guides/images/#asset-caching) for details. 
+
+Special thanks to [@oliverlynch](https://github.com/oliverlynch), who contributed this feature!
```

Comment:
```suggestion
Special thanks to [@oliverlynch](https://github.com/oliverlynch), who contributed to this feature!
```

Reply:
The meaning is that the feature was oliverlynch's contribution

Reply:
@yanthomasdev can you resolve this?

Reply:
I will resolve on behalf of docs! I think it's fine as-is to imply ownership of the feature! Yan can take it up with me in the parking lot after class...